cmd/enry/main.go

```diff
@@ -178,9 +182,9 @@ func (e filelistError) Error() string {
 	return fmt.Sprintf("Could not process the following files:\n%s", strings.Join(e, "\n"))
 }
 
-func printPercents(fSummary map[string][]string, buff *bytes.Buffer, mode string) {
+func printPercents(root string, fSummary map[string][]string, buff *bytes.Buffer, mode string) {
 	// Select the way we quantify 'amount' of code.
-	var reducer func([]string) (float64, filelistError)
+	var reducer func(string, []string) (float64, filelistError)
```

Comment:
Arguably out of scope and thus optional, but I recommend simplifying to:

```go
reducer := fileCountValues
switch mode {
  …
  // drop default case
}
```

Having the default explicitly set up front reduces type-declaration noise and makes it more obvious _which_ is the default without having to read down to that case. YMMV.